types/youtube/index.d.ts

```diff
@@ -518,6 +535,11 @@ declare namespace YT
 		 * Comma separated list of video IDs to play after the URL path's video.
 		 */
 		playlist?: string;
+		
+		/**
+		 * Whether videos play inline or fullscreen in an HTML5 player on iOS. (currently by default, Fullscreen).
+		 */
+		playsinline?: PlaysInline;
```

Comment:
@DanielRosenwasser But this is the convention already present in this file. It's the exact same as used for Loop, ShowInfo, ModestBranding, etc.